lib/segment/src/data_types/tiny_map.rs

```diff
@@ -1,12 +1,21 @@
 use tinyvec::TinyVec;
 
-const CAPACITY: usize = 3;
-
-pub struct TinyMap<K: Default, V: Default> {
-    list: TinyVec<[(K, V); CAPACITY]>,
+pub const CAPACITY: usize = 3;
+
+#[derive(Clone, PartialEq)]
```

Comment:
I am afraid we can't derive `PartialEq` here, underlying list will be order sensitive, which is incorrect for the map. I would propose to make custom eq by looking all keys from one man in another + checking the length

Reply:
You are right, will implement trait manually